src/lib/shopping-fpti/event-handlers.js

```diff
@@ -1,3 +1,4 @@
+import { capturePageData } from '../tag-parsers/capture-page-data';
 /* @flow */
```

Comment:
/* @flow */   - should be in first line of the file

Reply:
`capturePageData` isn't flow typed because it deals with parsers which deal with HTML dom nodes that are a pain to type in flow. So the flow comes after it's import.